webapp/src/components/channel_header_dropdown_button/index.ts

```diff
@@ -16,7 +16,7 @@ import {
 import ChannelHeaderDropdownButton from './component';
 
 const mapStateToProps = (state: GlobalState) => ({
-    show: isVoiceEnabled(state),
+    show: Boolean(channelState(state, getCurrentChannelId(state))?.enabled),
```

Comment:
re: the comment I just left about making a `isVoiceEnabled` selector. That would simplify this to:
```suggestion
    show: isVoiceEnabled(state, getCurrentChannelId(state)),
```
And in the other places you've used it.